src/generate-from-template.js

```diff
@@ -55,9 +57,11 @@ const generateFromTemplate = async ({ templateName, options = {} }) => {
       templateBanner = templateBannerResponse;
     }
   }
+  await buildLogger.addStep(1, 'complete');
```

Comment:
Same thing throughout, do these need to be awaited, since none of the functions are async?